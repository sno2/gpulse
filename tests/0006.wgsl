const a = 0x123;
const b = 0X123u;
const c = 0x3f;
