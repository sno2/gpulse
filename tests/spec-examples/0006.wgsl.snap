ast.Node{ .const_decl = ast.ConstDecl{ .name = ast.Span{ .start = 6, .end = 7 }, .typ = null, .value = ast.Node{ .number_literal = ast.NumberLiteral{ .span = ast.Span{ .start = 10, .end = 15 }, .kind = ast.NumberLiteral.NumberKind.abstract_int } } } }
ast.Node{ .const_decl = ast.ConstDecl{ .name = ast.Span{ .start = 23, .end = 24 }, .typ = null, .value = ast.Node{ .number_literal = ast.NumberLiteral{ .span = ast.Span{ .start = 27, .end = 33 }, .kind = ast.NumberLiteral.NumberKind.u32 } } } }
ast.Node{ .const_decl = ast.ConstDecl{ .name = ast.Span{ .start = 41, .end = 42 }, .typ = null, .value = ast.Node{ .number_literal = ast.NumberLiteral{ .span = ast.Span{ .start = 45, .end = 49 }, .kind = ast.NumberLiteral.NumberKind.f32 } } } }
